bgw_ccd.md

```diff
@@ -0,0 +1,12 @@
+---
+title: BGW/CCD Protocol
+parent: Protocols
+has_children: true
+nav_order: 3
+---
+
+# BGW/CCD Protocols
+
+In the case of threshold secret sharing with `t<n/2` (resp. `t<n/3`) one can obtain semi-honest (resp. robust actively) secure multi-party computation protocols over a finite field of `p` elements with `p>n` using Shamir Sharing and the BGW/CCD protocol. The protocol family can be extended to an arbitrary `Q2` (resp. `Q3`) access structures in a relatively straight forward manner. The protocol is interesting as it can protect against adversaries with infinite computing power.
```

Comment:
- circular: in explaining what is BGW/CCD you use BGW/CCD
- too technical : what is threshold secret sharing, what are `t` and `n`. finite field? Q2/Q3? 
- require links to semi-honest / active (malicious) articles